app/views/spree/admin/users/_form.html.haml

```diff
@@ -1,26 +1,26 @@
 .row
   .alpha.five.columns
     = f.field_container :email do
-      = f.label :email, Spree.t(:email)
+      = f.label :email, t(".email")
```

Comment:
sorry, what was the difference? I recall having a discussions about it but I don't remember.

Reply:
the main diff here is the lazy loading instead of using en.yml root entries.
the Spree.t thing, from @mkllnk 's wisdom through slack:

"I just looked up what the difference is: `core/lib/spree/i18n.rb`
    
```
def translate(*args)
      @virtual_path = virtual_path

      options = args.extract_options!
      options[:scope] = [*options[:scope]].unshift(:spree)
      args << options
      super(*args)
    end
    alias_method :t, :translate
```

So we can use `Spree.t` if we want to translate something in the Spree namespace. For example `Spree.t('settings')` is the same as `t('spree.settings')`. But we want to get away from all Spree translations in the long term."

the conclusion is that we should not use Spree.t at all.

Reply:
I have added this here: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development
under "Add missing Spree translations"